eng/pipelines/windows.yml

```diff
@@ -126,12 +126,12 @@ jobs:
 
         - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
           - netcoreappWindowsQueues: Windows.7.Amd64.Open+Windows.81.Amd64.Open+Windows.10.Amd64.ClientRS4.ES.Open
-          - nanoQueues: Windows.10.Nano.Amd64.Open
+          - nanoQueues: "`(Windows.Nano.1803.Amd64.Open`)windows.10.amd64.serverrs4.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1803-helix-amd64-05227e1-20190509225944"
```

Comment:
None of the quotes are necessary here. In the linux yml file we escape the `(` with `\`

Reply:
The ` chars definitely are since you use powershell but I can try it again with no "...  looks like there are some actual issues with the tests on System.Net.Sockets.Tests on newer Nanos (this is the first time running > 1709).

Reply:
Tried that combo, got "Pipeline configuration error:
/eng/pipelines/windows.yml: (Line: 129, Col: 25, Idx: 4219) - (Line: 129, Col: 26, Idx: 4220): While scanning for the next token, find character that cannot start any token." as seen in https://github.com/dotnet/corefx/pull/37632/checks?check_run_id=127527912 .  I assure you both quotes and back-ticks are needed here because you pass this parameter on a powershell command line.

Reply:
Understood. It seems like worth a comment?